src/components/splash/Splash.tsx

```diff
@@ -9,7 +9,7 @@ interface ISplash {
 
 export const Splash = (props: ISplash) => {
```

Comment:
also here, the object destructuring might be done (depends, which way you prefer)

```
interface ISplash {
  sectionBgColor: string;
  title: string;
}

export const Splash = ({ sectionBgColor, title }: ISplash) => {
  return (
    <SplashWrapper backgroundColor={sectionBgColor}>
      <SplashH6>{title}</SplashH6>
    </SplashWrapper>
  );
};
```